src/javascript/app/base/client.js

```diff
@@ -395,13 +395,7 @@ const Client = (() => {
 
     const hasCostaricaAccount = () => !!(getAllLoginids().find(loginid => /^CR/.test(loginid)));
 
-    const canRequestProfessional = () => {
-        const residence = get('residence');
-        /* Austria, Italy, Belgium, Latvia, Bulgaria,	Lithuania, Croatia, Luxembourg, Cyprus, Malta, Czech Republic,	Netherlands, Denmark, Poland, Estonia, Portugal, Finland, Romania, France, Slovakia, Germany, Slovenia, Greece, Spain, Hungary, Sweden, Ireland, United Kingdom, Australia, New Zealand, Singapore, Canada, Switzerland */
-        const countries = ['at', 'it', 'be', 'lv', 'bg', 'lt', 'hr', 'lu', 'cy', 'mt', 'cf', 'nl', 'dk', 'pl', 'ee', 'pt', 'fi', 'ro', 'fr', 'sk', 'de', 'si', 'gr', 'es', 'hu', 'se', 'ie', 'gb', 'au', 'nz', 'sg', 'ca', 'ch'];
-        return countries.indexOf(residence.toLowerCase()) !== -1;
-
-    };
+    const canRequestProfessional = () => State.getResponse('landing_company.financial_company.shortcode') === 'maltainvest';
```

Comment:
is the function name not misleading? now it's just checking that financial company's shortcode is maltainvest, so do we even need a function for that? why not just use `Client.get('landing_company_shortcode') === 'maltainvest'`, or use the `State.getResponse('landing_company.financial_company.shortcode') === 'maltainvest'` function in the one file that needs it?